### DISCUSSION TITLE
Is it possible to build/wrap the jsPanel over an element on the DOM?

### DISCUSSION THREAD

**Discussion by SmartPlugins** [Flyer53/jsPanel4#144] [Q&A]
How would I wrap/build the panel over a DOM element, that could be hosting a component?

**Comment by Flyer53** [comment]
Just to verify ... do you talk about an already existing element in the DOM you want to either clone or remove from the DOM and then use as content of the panel? If so, would this process happen with a click on the element in question (or something similar) or when creating a specific panel?

**Comment by Flyer53** [comment]
@SmartPlugins I won't be able to give you a proper answer unless you give me the requested information ... or even describe your request in a little bit more detail.
